packages/react-accordion/src/components/AccordionHeader/useAccordionHeader.tsx

```diff
@@ -54,7 +67,7 @@ export const useAccordionHeader = (props: AccordionHeaderProps, ref: React.Ref<H
     components: {
       root: 'div',
       button: 'button',
-      expandIcon: AccordionHeaderExpandIcon,
+      expandIcon: 'span',
```

Comment:
Why not using `ChevronRightRegular` as the component itself? do we really need to wrap it up into a span?

Reply:
A few reasons:
1. This is analogous to how it worked before: `AccordionHeaderExpandIcon` was not an actual icon; it was a component that rendered a span around an SVG.
2. This is how other components' icon slot works.
3. Based on how our slots work, there would be no way to change the icon without creating a custom AccordionHeader component. The only way to change the component is to set it in `state.components.expandIcon`.<br/> If I were to set it to e.g. `icon={<MyCustomIcon />}`, that would set the `children` of `ChevronRightRegular`, which doesn't work.